GameServerLib/Logic/GameObjects/GameObject.cs

```diff
@@ -43,7 +43,7 @@ public void SetTeam(TeamId team)
         public float CollisionRadius { get; set; }
         protected Vector2 _direction;
         public float VisionRadius { get; protected set; }
-        public bool IsDashing { get; protected set; }
+        public bool IsDashing { get; internal set; }
```

Comment:
It's bad practice to expose members like this outside of the class itself. Please add a `StopDashing`/`CancelDash`/`SetDashState` method instead.